### DISCUSSION TITLE
Depleted Fuel Rods do not stack to 32 like other crafting materials.

### DISCUSSION THREAD

**Discussion by Cherman0** [FakeFishGames/Barotrauma#13661] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Depleted fuel rods stored in a cabinet will stack to 8 and not 32 like other crafting materials. Expected behavior is that they behave like other crafting materials and stack to 32.

### Reproduction steps

1. Obtain more than 8 depleted fuel rods
2. Place them in a container
3. Observe that they will not fit in one stack like your other materials

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.2.13.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

This is actually intentional, all fuel rods only stack up to 8. I suppose depleted rods could be considered crafting materials and not "normal rods", in my opinion it makes sense to treat them the same as all other rods.

**Reply by Cherman0** [reply]
By "Depleted Fuel Rods" I'm referring to the craftable item from an engineer talent, not just fuel rods that are at 0 health. They arent usable as fuel rods at all and only serve as crafting materials, so it seems odd to have them behave like fuel rods. If it is intended that they behave like fuel rods and not like crafting materials, they should probably be placeable in the reactor, RFA, and mechsuit fuel slots.

**Reply by Regalis11** [reply]
Yeah, I assumed you meant the ones you can craft and use as materials. Even though they aren't usable as actual fuel, I personally see them as types of fuel rods regardless, and think it makes sense that they follow the same stacking rules as other rods for consistency. They are the same size and shape as normal rods, so to me it seems intuitive that they'd have the same restrictions for storage.

**Reply by NotWendy** [reply]
I agree with Regalis here. They look like fuel rods, so they stack like fuel rods.

**Comment by pavel4win** [comment]
We take a steel bar (from which you can make a whole shell for a railgun if you add iron)
uranium, which can power a boat for as long as 15 minutes.
lead, from which you can make a half of box for a coil gun 100 shots.
Then we burn this in the reactor.
add another steel bar there.
And two such "depleted fuels" are needed to harden a small screwdriver.
Where is the logic?
Where is the industrial waste?
How much will such a “screwdriver” weigh?

**Reply by NotWendy** [reply]
The logic lies purely in gameplay balance, nothing else. One unit of any raw material can be used to produce objects of varying sizes because nobody is interested in spending 100 units of steel per large item and 15 units of steel per small item, for the sake of accurately representing the item's mass. The unrealistic simplification is there to make inventory management less of a pain and make the game choke less when you collect a lot of materials.
With depleted fuel rods specifically, you're forced to spend multiple per item because of how common empty rods are.

**Reply by Regalis11** [reply]
@NotWendy well said.

Even though the use of materials is not realistic, I highly doubt most players would find it an improvement if you needed to manage materials by the gram or if the material requirements were based on realism, rather than what works for the in-game economy and gameplay.

I suppose one way to rationalize this is that the deconstructor isn't 100% efficient: fabricating anything creates waste materials that e.g. get dumped in the ocean or are stored somewhere in the deconstructor and periodically emptied in the outposts. For example, in the case of hardened screwdrivers, I wouldn't imagine the whole rod being used, but just the small amounts of heavy elements in it.

**Comment by Regalis11** [comment]
Discussed this with the rest of the team, and we've agreed not to change this for the reasons explained above.
